autogpt_platform/backend/backend/data/block.py

```diff
@@ -492,6 +510,135 @@ def get_blocks() -> dict[str, Type[Block]]:
     return load_all_blocks()
 
 
+def is_block_provider_configured(block_cls: type["Block"]) -> bool:
```

Comment:
Using `Block` without type arguments leaves `Block.input_schema` and `Block.output_schema` undefined
```suggestion
def is_block_provider_configured(
    block_cls: type["Block[BlockSchema, BlockSchema]"]
) -> bool:
```